src/app/components/MeasurementExplorer.tsx

```diff
@@ -37,7 +37,15 @@ export function MeasurementExplorer(props: MeasurementExplorerProps) {
     ...defaultInfo(0),
   });
   return (
-    <div style={{ width, height }}>
+    <div
+      style={{ width, height }}
+      css={css`
+        display: flex;
+        flex-direction: column;
+        gap: 40px;
+        justify-content: space-around;
+      `}
```

Comment:
Can you put all the styles either in `style` or in `css`, instead of having a mix of both?

Reply:
Also, do you have an explanation for the bug and why this resolves it?

Reply:
> do you have an explanation for the bug and why this resolves it?

MeasurementExplorer contain MeasurementPlot + select inputs (where we can change variables)
MeasurementPlot was taking 100% of it, but inputs have also a height so each time the page is rendring the plot change height ..etc (so it keeps stretching)

as a solution i used flex-col to divide MeasurementExplorer

Reply:
> Can you put all the styles either in `style` or in `css`, instead of having a mix of both?

It will be  
```
width: ${typeof width === 'number' ? `${width}px` : width};
height: ${typeof height === 'number' ? `${height}px` : height};
```
in css

Reply:
Alright you can leave it like it is then